editions/tw5.com/tiddlers/concepts/CoordinateSystems.tid

```diff
@@ -0,0 +1,42 @@
+created: 20220810201659784
+modified: 20220810201659784
+tags: Concepts
+title: Coordinate Systems
+type: text/vnd.tiddlywiki
+
+TiddlyWiki (primarily the RevealWidget) supports two coordinate systems for positioning popups (see PopupMechanism to learn more about popups).
+
+Beware that absolute coordinates were added with TiddlyWiki 5.2.4 and that they may not work with all extensions and plugins. For maximum backwards compatibility, use absolute coordinates only where necessary.
```

Comment:
We have a `.from-version` macro to introduce info, that works from version xxx
eg:

>`<<.from-version "5.2.4">>` We introduced absolute coordinates that work with all extensions and plugins. For maximum backwards compatibility, use absolute coordinates only where necessary.